addon/components/o-s-s/private/base-dropdown.ts

```diff
@@ -3,11 +3,14 @@ import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
 import { guidFor } from '@ember/object/internals';
 import { isTesting } from '@embroider/macros';
+import type { FeedbackMessage } from '../input-container';
```

Comment:
nit

as we use this type in more places, what about having a `addon/types` file (w/ its match file in `app`) that exports this ? 

(just for the new ones, not necessarily the one in input-container until places importing it are updated too)